plugin/rollback.go

```diff
@@ -0,0 +1,75 @@
+package plugin
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/mitchellh/mapstructure"
+)
+
+const (
+	rotateCredentialWAL = "rotateCredentialWAL"
+)
+
+// rotateCredentialWAL is used to store information in a WAL that can retry a
```

Comment:
should this comment be `rotateCredentialWAL` -> `rotateCredentialEntry`?

Reply:
Yes, good catch! I did some renaming and missed this 😄